static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-limited-impressions.js

```diff
@@ -146,21 +152,29 @@ define([
         };
 
         var componentWriter = function (component) {
-            fastdom.write(function () {
-                var submetaElement = $('.submeta');
-
-                if (submetaElement.length > 0) {
-                    component.insertBefore(submetaElement);
-                    embed.init();
-                    mediator.emit('contributions-embed:insert', component);
+            ajax({
+                url: 'https://api.nextgen.guardianapps.co.uk/geolocation',
```

Comment:
probably want to use `config.page.ajaxUrl` here rather than hardcode the endpoint